src/index.ts

```diff
@@ -1,6 +1,6 @@
-import { createElement, useRef, useEffect, useCallback, useMemo } from 'react'
+import { createElement, useRef, useEffect, useCallback, useMemo, HTMLAttributes } from 'react'
 
-interface Props {
+interface Props extends HTMLAttributes<HTMLElement>{
```

Comment:
**suggestion:** Can we save on an import and use the global namespace here?

```suggestion
interface Props extends React.HTMLAttributes<HTMLElement>{
```

Reply:
Done